spec/scenic/statements_spec.rb

```diff
@@ -108,17 +108,6 @@ module Scenic
           .with(:name, definition.to_sql)
       end
 
-      it "fails to replace the materialized view in the database" do
```

Comment:
Yeah I don't think we can get this in without coverage. Changing the deleted spec to exercise the new path successfully would cover it I think.